### ISSUE TITLE
RD does not start properly using dockerd container runtime

### ISSUE THREAD

**Issue by evertonlperes** [rancher-sandbox/rancher-desktop#1650]
### Actual Behavior

After a fresh install on Windows 10 PRO, selecting `dockerd` container runtime throws an error:

```bash
Error: wsl.exe exited with code 1
```
```bash
      exec /usr/local/bin/wsl-service local start
        ...
2022-02-24T16:41:34.369Z: Running command wsl --distribution rancher-desktop --exec wslpath -a -u C:\Users\everton.peres\AppData\Local\rancher-desktop\logs...
2022-02-24T16:41:34.495Z: Running command wsl --distribution rancher-desktop --exec wslpath -a -u C:\Users\EVERTO~1.PER\AppData\Local\Temp\rd-k3s-Z60cYk\k3s...
2022-02-24T16:41:34.614Z: Running command wsl --distribution rancher-desktop --exec cp /mnt/c/Users/EVERTO~1.PER/AppData/Local/Temp/rd-k3s-Z60cYk/k3s /etc/conf.d/k3s...
2022-02-24T16:41:34.732Z: Running command wsl --distribution rancher-desktop --exec chmod 644 /etc/conf.d/k3s...
2022-02-24T16:41:34.846Z: Running command wsl --distribution rancher-desktop --exec /sbin/rc-update --update...
2022-02-24T16:41:35.027Z: Running command wsl --distribution rancher-desktop --exec /usr/local/bin/wsl-service k3s start...
2022-02-24T16:41:35.151Z: WSL: executing: /usr/local/bin/wsl-service k3s start: Error: wsl.exe exited with code 1
```

`docker ps`
```bash
error during connect: In the default daemon configuration on Windows, the docker client must be run with elevated privileges to connect.: Get "http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.24/containers/json": open //./pipe/docker_engine: The system cannot find the file specified.
```

### Steps to Reproduce

1. Uninstall any older RD versions
2. Download RD dev [package](https://github.com/rancher-sandbox/rancher-desktop/actions/runs/1891215203): `1.0.0-341-gc178225`
3. Install it and select `dockerd` as a container runtime.
4. (optional step): Factory Reset the app
5. It should restart the k8s service
6. Should throw an error

### Result

![Screen Shot 2022-02-24 at 10 41 52 AM](https://user-images.githubusercontent.com/37411123/155569032-5e2ab13a-6743-4d73-8a37-b14b0eeb91fe.png)

### Expected Behavior

Selecting `dockerd` should reset vm/k8s and set up the `dockerd` container runtime properly.

### Additional Information

All logs are empty:
![Screen Shot 2022-02-24 at 10 48 16 AM](https://user-images.githubusercontent.com/37411123/155569376-07c08a3e-5dab-4ead-802d-b00c4b64e3cf.png)

### Rancher Desktop Version

 1.0.0-341-gc178225

### Rancher Desktop K8s Version

1.22.6

### Which container runtime are you using?

moby (docker cli)

### What operating system are you using?

Windows

### Operating System / Build Version

Windows 10 PRO 20H2

### What CPU architecture are you using?

x64

### Windows User Only

N/A

**Comment by mattfarina** [comment]
Seems we both filed an issue. More information on https://github.com/rancher-sandbox/rancher-desktop/issues/1651

**Comment by jandubois** [comment]
I don't see any information in here that makes me think it isn't the same as #1651, which should be fixed by #1653.
